src/app/utils/custom-errors.js

```diff
@@ -12,7 +12,16 @@ class WebhookValidationError extends Error {
   }
 }
 
+class AxiosError extends Error {
```

Comment:
I think Axios has it's own error, no? Is there a reason why we don't use Axios's own `AxiosError`

Reply:
it only has a type, can't seem to find the actual class. Moved the custom class to the test file instead of the util, since it is not used anywhere else.